src/Nox.Reference.Data.Machine/README.md

```diff
@@ -1,63 +1,9 @@
-# Nox.Reference.MacAddresses
-MacAddresses list and info
+In order to create migrations:
```

Comment:
Tell what this project is needed for, don't just add message about migration please

Reply:
#18 
/